game-core/src/main/java/games/strategy/engine/data/RouteFinder.java

```diff
@@ -16,27 +17,29 @@
 import java.util.Set;
 import java.util.function.Function;
 import java.util.function.Predicate;
+import javax.annotation.Nullable;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor(access = AccessLevel.PACKAGE, onConstructor_ = @VisibleForTesting)
```

Comment:
As far as I can tell this constructor is only used by this class, so it might as well be private, without the annotation.

Reply:
Yeah, this one went in circles. I was expecting to need to be able to control `MoveValidator` for test, though it appears it's not covered by the test cases and therefore injecting a mock is not necessary.